HISTORY.md

```diff
@@ -12,6 +12,7 @@
 * Fix a bug where range tombstone blocks in ingested files were cached incorrectly during ingestion. If range tombstones were read from those incorrectly cached blocks, the keys they covered would be exposed.
 * Fix a data race that might cause crash when calling DB::GetCreationTimeOfOldestFile() by a small chance. The bug was introduced in 6.6 Release.
 * Fix a bug where a boolean value optimize_filters_for_hits was for max threads when calling load table handles after a flush or compaction. The value is correct to 1. The bug should not cause user visible problems.
+* Fix a bug in FIFO compaction with TTL. Before this fix and since RocksDB 6.8, FIFO compaction can drop a file whose oldest key has expired while others have not. This fix also requires that max_open_files = -1.
```

Comment:
I suspect that the word "expire" can confuse some people. Can we just explain the behavior before, since 6.8 and after the fix? Also word "bug" may confuse people too. Maybe just say "behavior change" or something like that.